src/edge-core-index.js

```diff
@@ -397,6 +397,7 @@ export type EdgeAccount = {
   getFirstWalletInfo(type: string): ?EdgeWalletInfo,
   getWalletInfo(id: string): EdgeWalletInfo,
   listWalletIds(): Array<string>,
+  listSplittableWalletTypes(walletId: string): Array<string>,
```

Comment:
We still need to add this function to the `EdgeCurrencyPlugin` type.

Reply:
added